src/app/modules/core/components/loading-overlay/loading-overlay.component.html

```diff
@@ -0,0 +1,11 @@
+<div *ngIf="uploading">
+    <div class="text-white text-xl mt-4">
+      Uploading files...
+    </div>
+    <div class="my-4 text-hint text-lg leading-relaxed">
+      Hang in there while we upload your keystore files...
```

Comment:
```suggestion
      Hang in there while we upload your files...
```
We could use this as an overlay for all kinds of things, not just keystores in the future